client/src/periodUtils.js

```diff
@@ -31,17 +35,28 @@ export const PERIOD_FACTORIES = {
     start: date.clone().subtract(offset, "weeks").startOf("week"),
     end: date.clone().subtract(offset, "weeks").endOf("week")
   }),
-  // FIXME: biweekly calculation should be changed, first agree on what it means
-  [RECURRENCE_TYPE.BIWEEKLY]: (date, offset) => ({
-    start: date
-      .clone()
-      .subtract(2 * offset, "weeks")
-      .startOf("week"),
-    end: date
+  [RECURRENCE_TYPE.BIWEEKLY]: (date, offset) => {
+    // every biweekly period's start is even number of weeks apart from reference monday
+    const refMonday = moment(refMondayForBiweekly).startOf(weekType)
+    const curWeekMonday = date.startOf(weekType)
```

Comment:
```suggestion
    const curWeekMonday = date.clone().startOf(weekType)
```
or maybe
```suggestion
    const curWeekMonday = moment(date).startOf(weekType)
```
In this file we use the .clone() way, but in the rest of ANET we used most of the time the moment(date) way.